src/app/components/app-layout/app-layout.component.ts

```diff
@@ -33,11 +33,12 @@ export class AppLayoutComponent implements OnInit {
   public isAuthenticated: boolean;
   public subscription: Subscription;
   public displayAlertContainer = false;
+  public contentOnlyLayout = false;
```

Comment:
Property could have a better name, without context I would not know what this property is used for. Suggested: displayContentWithoutLayout